spec/inputs/beats/message_listener_spec.rb

```diff
@@ -199,7 +199,7 @@ def flush(&block)
       end
     end
 
-    context "when the message is from LSF" do
+    context "when the message is from Lumberjack" do
       let(:message) { MockMessage.new("abc", { "line" => "hello world", '@metadata' => {} } )}
```

Comment:
review note: we keep unit test against Lumberjack and if we need integration tests we better place in [the core](https://github.com/elastic/logstash/blob/main/qa/integration/specs/beats_input_spec.rb) but since we are not actively maintaining Lumberjack and pushing LS-LS comm with new integration plugin, I don't think it is mandatory now.